tests/unit/action/test_bonsai_asset.py

```diff
@@ -0,0 +1,243 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
Nitpick: I suggest renaming the directory to `tests/unit/actions` (note the plural), as we already use plural with `modules` and `module_utils` at the same level.

Reply:
I initially named this folder `actions`, but then I changed my mind and decided to keep it consistent with the name in the `plugins` directory where action plugins live in the `action` folder. And yes, that mix of singular and plural names is making me go crazy, but there is not much to do here since this is what Ansible prescribes ;)

Reply:
Oh, right, I see. I'm good then :+1: